docs/public/specifications/tests/Basic-TP/basic01.md

```diff
@@ -75,6 +75,11 @@ severity level can be changed in the [Zonemaster-Engine profile]. Also see the
 The argument names in the Arguments column lists the arguments used in the
 message. The argument names are defined in the [argument list].
 
+The name server names are assumed to be available at the time when the msgid
+is created, if the argument name is "ns" or "ns_list" even when in the
+"[Test procedure]" below it is only referred to the IP address of the name
+servers.
```

Comment:
I’m skeptical about including wording like this. I don’t think it’s a good idea to overload “IP address” to really mean “name and IP address pair”.

Reply:
That is not what is says. It does not say that "IP adress" means namn and IP. It says that the name is available when creating the messagesl. 

 The text is copied from the end of section https://github.com/zonemaster/zonemaster/blob/master/docs/public/specifications/tests/Nameserver-TP/nameserver15.md#summary

What is you suggestion? Reverting to ns_ip and ns_ip_list, respectively?